.github/workflows/ci.yml

```diff
@@ -152,7 +160,44 @@ jobs:
       - name: Archive profiler artifacts
         uses: actions/upload-artifact@v3
         with:
-          name: profiler-${{ matrix.artifact-name || matrix.os }}
+          name: profiler-${{ matrix.artifact-name }}${{ steps.version.output.suffix }}
           path: |
             target/release/profiler${{ matrix.os == 'windows-latest' && '.exe' || '' }}
           if-no-files-found: error
+
+  release:
+    name: Publish release
+    if: github.event_name == 'push' && github.ref_type == 'tag'
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+        with:
+          path: ./artifacts
+      - name: List artifacts
+        run: ls -R ./artifacts
+      - name: Create a draft release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          draft: true
```

Comment:
Why prerelease and draft? Imo it could be just a release. Wdyt?

Reply:
I'm OK with having it a draft first, in case we want to check if the release is OK before we publish it, or in case we just need it for testing and not actually want to publish it